wherehows-web/app/constants/datasets/compliance.ts

```diff
@@ -79,6 +71,14 @@ enum ComplianceFieldIdValue {
   SlideshareUserId = 'SLIDESHARE_USER_ID'
 }
 
+/**
+ * Defines the string values for a custom id logical tyoe
```

Comment:
I think you have a `tyoe-po` here :trollface:

Reply:
hahaha, nice! Love the attention to detail 🥇 🍾  @ttbach